src/Sylius/Bundle/PaymentBundle/Resources/config/app/state_machine.yml

```diff
@@ -14,7 +14,7 @@ winzou_state_machine:
             completed: ~
             failed: ~
             cancelled: ~
-            void: ~
+            unknown: ~
```

Comment:
Should this be mentioned in UPGRADE? Though realistically it won't affect anyone who uses ful sylius/sylius and reads UPGRADE.

Reply:
#15422